grafana/README.md

```diff
@@ -0,0 +1,14 @@
+# Grafana dashboard for mgmt
```

Comment:
Want to add a link to this file from README.md in the docs/ section?

I think this README should me in docs/ instead. Are you okay with that?